.circleci/config.yml

```diff
@@ -190,6 +192,8 @@ executors:
       NODE_ENV: development
       CUSTOMS_SERVER_URL: none
       HUSKY_SKIP_INSTALL: 1
+      # use apiKeys mode and real numbers for smoke tests
+      RECOVERY_PHONE__TWILIO__CREDENTIAL_MODE: 'apiKeys'
```

Comment:
When we run the smoke the smoke tests, it's just running the playwright tests against our servers. There shouldn't be any 'server side' configuration required. But the  `RECOVERY_PHONE__TWILIO__CREDENTIAL_MODE` env is specific to server side configuration, so I don't _think_ it is necessary.

Reply:
What is important, is that `FUNCTIONAL_TESTS__TWILIO__TEST_NUMBER__STAGE` and `FUNCTIONAL_TESTS__TWILIO__TEST_NUMBER__PROD` is set. This value is secrete, and I believe this has been configured in the circleci env config, but we can double check.

Reply:
@dschom We have `FUNCTIONAL_TESTS__TWILIO__TEST_NUMBER` set in CircleCi, no distinction between stage and prod for any vars.